packages/platform-test-suite/lib/test/createClientWithFundedWallet.js

```diff
@@ -52,10 +52,10 @@ async function createClientWithFundedWallet(HDPrivateKey = undefined) {
     wallet: walletOptions,
   });
 
-  const amount = 40000;
-
   await fundWallet(faucetClient.wallet, client.wallet, amount);
 
+  await faucetClient.wallet.disconnect();
```

Comment:
I am afraid it will increase time for devnet and testnet significanlty.

Reply:
From my calculations, it's a 3% time increase in the worst case:
- It takes around 10-15 seconds to init the faucet wallet from storage on testnet. 
- The faucet wallet is used about 6-7 times in the test suite, hence increasing the overall running time to 105 seconds in the worst case. 
- The overall runtime of the platform test suite on testnet is around 1 hour currently

Reply:
Oh, so it reuses the state from the previous run and doesn't sync the whole thing every time. Am I right?

Reply:
Yep. That's the whole point of embedding storage into the faucet wallet :)